cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,49 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/alerts');
   });
 
-  it('should have the ability to assert automatically resolved alerts', () => {
-
+  it('1should have the ability to assert automatically resolved alerts', () => {
+    cy.get('#alertButton').click();
+    cy.on('window:alert', (alerts) => {
+      expect(alerts).to.equal('You clicked a button');
+    });
   });
 
-  it('should have the ability to assert scheduled allert', () => {
-
+  it('2should have the ability to assert scheduled allert', () => {
+    cy.get('#timerAlertButton').click();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(5000);
+    cy.on('window:alert', (alerts) => {
+      expect(alerts).to.equal('This alert appeared after 5 seconds');
+    });
   });
 
-  it('should autimatically resolve alerts', () => {
+  it('3should autimatically resolve alerts', () => {
+    cy.get('#confirmButton').click();
+    cy.on('window:alert', (alerts) => {
+      expect(alerts).to.equal('Do you confirm action?');
+    });
 
+    cy.get('#confirmResult').should('You selected Ok');
```

Comment:
You need to check your tests before sending them to review again
![image](https://github.com/mate-academy/qa_cypress_e2e_alerts/assets/85580286/b0780624-45b3-4e3d-a0c5-f680c044c1a4)